### PR TITLE
Camera control

### LINKED ISSUE #6: Create camera and implement its control
Implement the camera that have pan and zoom

### PR DESCRIPTION
**Pull request by AMIOL7**
Here is my implementation of camera control. Please review it

### CODE REVIEW

File: app/src/main/java/ELME/App.java
Comment (vismate) [inline app/src/main/java/ELME/App.java]: Maybe remove comments?

File: app/src/main/java/View/CustomWheelListener.java
Comment (vismate) [inline app/src/main/java/View/CustomWheelListener.java]: Remove license template

File: app/src/main/java/View/CustomWheelListener.java
Comment (vismate) [inline app/src/main/java/View/CustomWheelListener.java]: Camera does not need to be stored. ```Game.world.camera()``` can get you the current camera.

File: app/src/main/java/View/MainScreen.java
Comment (vismate) [inline app/src/main/java/View/MainScreen.java]: I recommend not opening an image at all, because we don't have any images stored in the repo. Also this code opens the image EVERY FRAME!

File: app/src/main/java/View/MouseDraggedListener.java
Comment (vismate) [inline app/src/main/java/View/MouseDraggedListener.java]: Debug prints should be removed.

**Review by vismate — CHANGES_REQUESTED** [review]

**Comment by AMIOL7** [comment]
Your comments are received will work on them a bit later

**Comment by AMIOL7** [comment]
@vismate Your comments have been resolved. Please review again

**Comment by AMIOL7** [comment]
Changes based on @vismate's comments have been made. Please review again

File: app/src/main/java/ELME/App.java
Comment (vismate) [inline app/src/main/java/ELME/App.java]: This comment could be removed

File: app/src/main/java/ELME/App.java
Comment (vismate) [inline app/src/main/java/ELME/App.java]: Debug prints could be removed.

File: app/src/main/java/ELME/View/CustomWheelListener.java
Comment (vismate) [inline app/src/main/java/ELME/View/CustomWheelListener.java]: Comment could be removed

File: app/src/main/java/ELME/View/MainScreen.java
Comment (vismate) [inline app/src/main/java/ELME/View/MainScreen.java]: License template could be removed

File: app/src/main/java/ELME/View/MouseDraggedListener.java
Comment (vismate) [inline app/src/main/java/ELME/View/MouseDraggedListener.java]: License template could be removed

File: app/src/main/java/ELME/View/CustomWheelListener.java
Comment (vismate) [inline app/src/main/java/ELME/View/CustomWheelListener.java]: If camera changes after construction, the zoom functionality will not work. I recommend not storing a reference to a camera all together. Game.world().camera() could be used in method to set zoom.

**Review by vismate — APPROVED** [review]

**Comment by vismate** [comment]
About my latest comments: I admit, I am a bit nit-picky. I do apologize, I don't want to block progress, so we can merge regardless.  I could also make these changes myself, if you feel like it.

**Comment by AMIOL7** [comment]
@vismate Already done myself.

**Comment by AMIOL7** [comment]
@pszi and @bgdxqi Please review this pull request

**Comment by vismate** [comment]
> @vismate Already done myself.

Fantastic! Thank you!

**Comment by vismate** [comment]
Would you like to merge now, or we should wait for other reviews?

**Comment by AMIOL7** [comment]
I'd wait for at least 1 more review, but if it can't wait  we can merge now

**Comment by vismate** [comment]
I'd like to start working on things that requires this branch. But I agree, one more person should review it at least. @pszi @bgdxqi Could one of you review it tomorrow, please?

**Review by bgdxqi — APPROVED** [review]
I might be not the most fit to review this. But to me it looks fine :D

**Comment by vismate** [comment]
Then this branch should be merged, in my oppinion.

**Comment by AMIOL7** [comment]
I am merging it now

**Pull request merged**
